src/sections/bt-background-image.liquid

```diff
@@ -32,8 +32,8 @@
     </div>
     {% schema %}
     {

```

Comment:
the bt-"section-name" is totally my fault. Let's make the name something more friendly to the user. the class can keep the prefix bt if it's not displayed to the user (I'm not sure about that).